test/signaturelistTest.js

```diff
@@ -1,4 +1,4 @@
-const assert = require('chai').assert;
+/*const assert = require('chai').assert;
```

Comment:
@jruiz50 Can you figure out why we can't use the "Import statement like ES6 in our project"?